util/optional_mutex.c

```diff
@@ -51,6 +52,7 @@ optional_mutex_unlock(pthread_mutex_t * mutex)
 {
 
 #ifdef OPTIONAL_MUTEX_PTHREAD_YES
+	/* Done! */
```

Comment:
This (and the preceding one) could be made more useful e.g. `/* Mutexes are enabled; call the pthread function. */`.